nestor_api/lib/k8s/builders.py

```diff
@@ -41,6 +64,143 @@ def load_templates(templates_path: str, template_names: list) -> dict:
     }
 
 
+def get_sections_for_process(
+    process: dict, deployment_config: dict, tag_to_deploy: str, templates: dict
+) -> list:
+    """Build the deployment sections of a process."""
+    process_name = process["name"]
+    app_name, sanitized_process_name, metadata_name = get_sanitized_names(
+        deployment_config, process_name
+    )
+    image_name = get_image_name(deployment_config, {"tag": tag_to_deploy})
+
+    deployment_sections = []
```

Comment:
According to this : https://kubernetes.io/docs/concepts/cluster-administration/manage-deployment
```suggestion
    deployment_resources = []
```